src/honey/sql.cljc

```diff
@@ -161,13 +161,12 @@
 (def ^:no-doc ^:dynamic *escape-?* true)
 
 ;; suspicious entity names:
-(def ^:private suspicious #";")
-(defn- suspicious? [s] (boolean (re-find suspicious s)))
+(defn- suspicious? [s] (str/includes? s ";"))
 (defn- suspicious-entity-check [entity]
     (when-not *allow-suspicious-entities*
       (when (suspicious? entity)
         (throw (ex-info (str "suspicious character found in entity: " entity)
-                        {:disallowed suspicious})))))
+                        {:disallowed ";"})))))
```

Comment:
Now you have the "magic" string `";"` in two places -- this should still use a `^:private` `def` for the string.

At some point, it may need to trap more than a single character so it may have to go back to either a regex or maybe a vector of substrings.

Reply:
If that ever happens, it can go back to regular expressions (or something else). But for now, it makes sense to not overpay for what's not needed.